Collections/Map Utilities/move.alias

```diff
@@ -483,8 +502,15 @@ if c:
 
 mapStates = load_json(get('mapStates','{}'))
 prevBack = mapStates.get(chanid())
-mapStates.update({chanid(): out})
-set_uvar('mapStates', dump_json(mapStates))
+checkMS = str(mapStates)
+lengthTotal = len(checkMS) + len(out)
+if lengthTotal >= 10000:
+  delete_uvar("mapStates")
+  set_uvar('mapStates', dump_json(mapStates))
```

Comment:
So this definitely just sets the uvar to what it was, without actually updating it. It deletes it, then makes it the same as it was right before you deleted it.